### DISCUSSION TITLE
Ultrahand Overlay Discussions

### DISCUSSION THREAD

**Discussion by ppkantorski** [ppkantorski/Ultrahand-Overlay#10] [Announcements]
## 👋 Welcome to Ultrahand Overlay Discussions!

Ultrahand Overlay Discussions is the ideal platform to connect with fellow members of our community and delve into all things related to Ultrahand Overlay. Here's how you can participate:

* **Pose inquiries:** Don't hesitate to ask about any aspect of Ultrahand Overlay that piques your interest.
* **Share your insights:** If you have ingenious suggestions or exciting concepts, please share them with us!
* **Engage with your fellow community members:** Join discussions, offer valuable insights, and collaborate with your fellow community members.
* **Extend a warm welcome and foster open-mindedness:** Let's cultivate an environment that is welcoming and open-minded. Respect each other'sviewpoints, and let's work together to enhance our community's strength 💪.

To begin, please take a moment to introduce yourself and share your role within the Ultrahand Overlay community. We are eager to learn more about you and your contributions.

<!--
For project maintainers, here are some valuable tips 💡 to kickstart your Discussions. You can leave these as Markdown comments for now or feel free to remove the comments so that all maintainers can see them:

📢 **Announce to your community** that Discussions is now available! Share the news on Twitter, post it, or link it from your website to encourage participation.

🔗 If you use issue templates, remember to **link any relevant issue templates** such as questions and community conversations to Discussions. This will help you organize your issues more efficiently by directing community content to Discussions. If you need assistance, here's a [link to the documentation](https://docs.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser).

➡️ You can also **convert issues to discussions**, either individually or in bulk based on labels. Keep an eye out for issues labeled as "question" or "discussion" that you may want to move to Discussions.

**Comment by FunkSmugly** [comment]
Hi! I’m really enjoying the Ultrahand Overlay. I wanted to see if a fix would be implementable for an issue that has persisted with Tesla; display scaling (overscan). The Ultrahand Overlay will not show up in games if the display scaling of the console is not at 100%. Thank you for your many contributions!

**Reply by ppkantorski** [reply]
Hopefully you enjoy the underscan fix.  Glad you like the project. 🙏🏼

**Reply by FunkSmugly** [reply]
Yeah, I think it's a great enhancement to an already terrific project. Not only will it allow users to finally scale to their displays, but it will also eliminate most of the confusion when anyone tries scaling for the first time and has to determine why Tesla "stopped working".  It seems to have generated some buzz on Discord as well. Hopefully, most of the overlays will be updated using the libraries you've created. Huge thanks and best of luck with all of your endeavors!
